g4py.sh

```diff
@@ -0,0 +1,51 @@
+package: G4PY
+version: "%(tag_basename)s%(defaults_upper)s"
+tag: v10.3.1
+source: https://github.com/PMunkes/geant4
+requires:
+  - GEANT4
+  - Python-modules
+  - ROOT
+  - boost
+  - XercesC
+  - opengl
+build_requires:
+  - CMake
+  - "Xcode:(osx.*)"
+env:
+  G4PYINSTALL: "$G4PY_ROOT"
+---
+#!/bin/bash -e
+cp -a $SOURCEDIR/environments/g4py/* $INSTALLROOT
+cmake $INSTALLROOT                               \
```

Comment:
Unfortunately with `cmake $SOURCEDIR`G4PY installs itself into `$SOURCEDIR.` This seems to be intentional behaviour, and as such I am installing the minimal amount of source files necessary to build G4PY in `$INSTALLROOT` to install into the proper directory. Otherwise the package depends on the source directory to always be available.